src/stores/toolbarStore.ts

```diff
@@ -1,41 +1,76 @@
 import { observable, computed, action } from 'mobx';
+import _ from 'lodash';
 import ToolbarTool from '~/enums/toolbarTool';
 import EditMode from '~/enums/editMode';
 
+import BaseTool from '~/tools/BaseTool';
+import AddNetworkNodeTool from '~/tools/AddNetworkNodeTool';
+import AddNewRoutePathTool from '~/tools/AddNewRoutePathTool';
+import CopyTool from '~/tools/CopyTool';
+import DivideLinkTool from '~/tools/DivideLinkTool';
+import EditNetworkNodeTool from '~/tools/EditNetworkNodeTool';
+import PrintTool from '~/tools/PrintTool';
+
+const TOOLS = [
+    new AddNetworkNodeTool(),
+    new AddNewRoutePathTool(),
+    new CopyTool(),
+    new DivideLinkTool(),
+    new EditNetworkNodeTool(),
+    new PrintTool(),
+];
+
 export class ToolbarStore {
-    @observable private _activeTool: ToolbarTool;
+    @observable private _selectedTool: BaseTool|null;
```

Comment:
`_selectedTool?` rather? And undefined instead of null.

Reply:
I think null is better choice here because it is a valid value, "no tool selected". For me undefined would refer to something not being initialized

Reply:
Wouldn't undefined also mean that the current tool is undefined as in not set?

Reply:
Maybe we should all generally agree on this how to use null/undefined.